daft/expressions/expressions.py

```diff
@@ -703,6 +703,42 @@ def sum(self) -> Expression:
         expr = self._expr.sum()
         return Expression._from_pyexpr(expr)
 
+    def approx_count_distinct(self) -> Expression:
+        """
+        Calculates the approximate number of unique values in the expression.
+
+        Approximation is performed using the [`HyperLogLog`](https://en.wikipedia.org/wiki/HyperLogLog) algorithm.
+
+        There is an important thing to note here:
+
+        If you're calling `approx_count_distinct` on a column, the presence of a `NULL` value will *not* be counted towards the final result.
```

Comment:
This seems a bit verbose for the docstring. We should just have the title be
```
Calculates the approximate number of unique non-null values in the expression
```
since we also have a null in the example.

Reply:
Updated.